### PR TITLE
Kdesktop 1440 failed to normalize change log level from warning to debug

### PR DESCRIPTION
**Pull request by ChristopheLarchier**

### CODE REVIEW

File: src/gui/parametersdialog.cpp [hunk 1]
```diff
@@ -702,7 +702,7 @@ QString ParametersDialog::getInconsistencyText(InconsistencyType inconsistencyTy
         text += (text.isEmpty() ? "" : "\n");
         text +=
                 tr("The file/directory name contains a recent UNICODE character not yet supported by your filesystem.<br>"
```
Comment (luc-guyot-infomaniak) [inline src/gui/parametersdialog.cpp:704]:
```suggestion
                tr("The item name contains a recent UNICODE character not yet supported by your filesystem.<br>"
```

File: src/libcommonserver/utility/utility.cpp [hunk 2]
```diff
@@ -322,15 +322,15 @@ std::string Utility::fileSystemName(const SyncPath &targetPath) {
...
     if (GetVolumeInformation(targetPath.root_path().c_str(), NULL, 0, NULL, &dwMaxFileNameLength, &dwFileSystemFlags,
                              szFileSystemName, sizeof(szFileSystemName)) == TRUE) {
-        return Utility::ws2s(szFileSystemName);
+        return ws2s(szFileSystemName);
     } else {
         // Not all the requested information is retrieved
         DWORD dwError = GetLastError();
         LOGW_WARN(logger(),
-                  L"Error in GetVolumeInformation for " << Path2WStr(targetPath.root_name()).c_str() << L" (" << dwError << L")");
+                  L"Error in GetVolumeInformation for " << formatSyncName(targetPath.root_name()) << L" (" << dwError << L")");
```
Comment (luc-guyot-infomaniak) [inline src/libcommonserver/utility/utility.cpp:330]:
```suggestion
                  L"Error in GetVolumeInformation for " << formatSyncName(targetPath.root_name()) << L" (" << getErrorMessage(dwError) << L")");
```

File: src/libcommonserver/utility/utility.cpp [hunk 3]
```diff
@@ -646,10 +645,8 @@ bool Utility::normalizedSyncName(const SyncName &name, SyncName &normalizedName,
             DWORD dwError = GetLastError();
             if (dwError != ERROR_INSUFFICIENT_BUFFER) {
                 // Real error, not buffer error
-                LOGW_WARN(logger(), L"Failed to normalize string " << SyncName2WStr(name).c_str() << L" - error code: "
-                                                                   << std::to_wstring(dwError));
-                SentryHandler::instance()->captureMessage(SentryLevel::Fatal, "Utility::normalizedSyncName",
-                                                          "Failed to normalize string");
+                LOGW_DEBUG(logger(), L"Failed to normalize string " << formatSyncName(name) << L" - error code: "
+                                                                    << std::to_wstring(dwError));
```
Comment (luc-guyot-infomaniak) [inline src/libcommonserver/utility/utility.cpp:649]:
```suggestion
                                                                    << getErrorMessage(dwError));
```

File: src/libcommonserver/utility/utility.cpp [hunk 4]
```diff
@@ -660,10 +657,8 @@ bool Utility::normalizedSyncName(const SyncName &name, SyncName &normalizedName,
 
     if (iSizeEstimated <= 0) {
         DWORD dwError = GetLastError();
-        LOGW_WARN(logger(), L"Failed to normalize string " << SyncName2WStr(name).c_str() << L" - error code: "
-                                                           << std::to_wstring(dwError));
-        SentryHandler::instance()->captureMessage(SentryLevel::Fatal, "Utility::normalizedSyncName",
-                                                  "Failed to normalize string");
+        LOGW_DEBUG(logger(),
+                   L"Failed to normalize string " << formatSyncName(name) << L" - error code: " << std::to_wstring(dwError));
```
Comment (luc-guyot-infomaniak) [inline src/libcommonserver/utility/utility.cpp:661]:
```suggestion
                   L"Failed to normalize string " << formatSyncName(name) << L" (" << getErrorMessage(dwError) << L")");
```

File: src/libcommonserver/utility/utility.cpp (hunk 2 above)
Comment (ChristopheLarchier) [reply src/libcommonserver/utility/utility.cpp]: Done

File: src/libcommonserver/utility/utility.cpp (hunk 3 above)
Comment (ChristopheLarchier) [reply src/libcommonserver/utility/utility.cpp]: Done

File: src/libcommonserver/utility/utility.cpp (hunk 4 above)
Comment (ChristopheLarchier) [reply src/libcommonserver/utility/utility.cpp]: Done

File: src/libcommonserver/utility/utility.cpp [hunk 5]
```diff
@@ -680,6 +675,7 @@ bool Utility::normalizedSyncName(const SyncName &name, SyncName &normalizedName,
 
     if (!strResult) { // Some special characters seem to be not supported, therefore a null pointer is returned if the conversion
                       // has failed. e.g.: Linux can sometime send filesystem events with strange character in the path
+        LOGW_DEBUG(logger(), L"Failed to normalize string " << formatSyncName(name));
```
[the remark this answers is not in the document]
Comment (ChristopheLarchier) [reply src/libcommonserver/utility/utility.cpp]: Done

File: src/libcommonserver/utility/utility.cpp [hunk 6]
```diff
@@ -733,26 +729,26 @@ bool Utility::checkIfDirEntryIsManaged(const DirectoryEntry &dirEntry, bool &isM
...
+        LOGW_DEBUG(logger(), L"Error in IoHelper::getItemType: " << formatIoError(dirEntry.path(), ioError));
         return true;
     }
 
     isLink = itemType.linkType != LinkType::None;
     if (!dirEntry.is_directory() && !dirEntry.is_regular_file() && !isLink) {
-        LOGW_WARN(logger(), L"Ignore " << formatSyncPath(dirEntry.path()).c_str()
-                                       << L" because it's not a directory, a regular file or a symlink");
+        LOGW_WARN(logger(),
+                  L"Ignore " << formatSyncPath(dirEntry.path()) << L" because it's not a directory, a regular file or a symlink");
```
[the remark this answers is not in the document]
Comment (ChristopheLarchier) [reply src/libcommonserver/utility/utility.cpp]: Done

**Review by luc-guyot-infomaniak — APPROVED** [review]

**Review by herve-er — APPROVED** [review]

**Pull request merged**